source/clear-linux/tutorials/dlaas-benchmarks.rst

```diff
@@ -0,0 +1,186 @@
+.. _dlaas-benchmarks:
+
+Deep Learning as a Service Benchmarking
+#######################################
+
+This tutorial details the steps needed to run benchmarking workloads in |CL-ATTR| using TensorFlow* and on Kubeflow with the |CL| Deep Learning as a Service Stack.
+
+Prerequisites
+=============
+
+For this tutorial, we'll be using a system that has |CL| installed using the :ref:`bare-metal-install` getting started guide.  It is possible to follow these instructions on other OS systems but we're limiting the scope of this document to |CL|.
+
+Ensure you have the "containers-basic" bundle installed, this will provide Docker, which is required for TensorFlow benchmarking. Use the :command:`swupd` utility with the `bundle-list` option and check for "containers-basic" in the list:
+
+.. code-block:: bash
+
+  sudo swupd bundle-list
+
+If you need to install the containers-basic bundle, use :command:`swupd` to do so.
+
+.. code-block:: bash
+
+  sudo swupd bundle-add containers-basic
+
+
+To run Kubernetes on |CL|, please refer to the :ref:`kubernetes` tutorial to ensure it is correctly installed and configured.
+
+We have validated these steps against the following software package versions
+
+  * |CL| 26240
+  * Docker 18.06.1
+  * Kubernetes 1.11.3
+  * Go 1.11.12
+
+
+TensorFlow Benchmarks
+=====================
+
+This section describes running the `TensorFlow benchmarks`_ as run by the TensorFlow community. These steps provide a template to run other benchmarks, providing they can invoke TensorFlow.
+
+The |CL| Deep Learning Stack is available in two versions.  First, a version that includes TensorFlow* optimized for Intel Architecture, and second, a version that includes the TensorFlow* framework optimized using Intel® Math Kernel Library for Deep Neural Networks (Intel® MKL-DNN) primitives.
+
+Download and run either the `TensorFlow`_ or the `TensorFlow with MKL-DNN`_  docker image from hub.docker.com. The next commands will take place in the running container. Replace <docker_name> with the name of the image.
+
+First, clone the benchmark repository
+
+.. code-block:: bash
+
+  docker exec -t <docker_name> bash -c ‘git clone http://github.com/tensorflow/benchmarks -b cnn_tf_v1.11_compatible’
+
+Next, execute the benchmark script to run the benchmark
+
+.. code-block:: bash
+
+  docker exec -i <docker_name> bash -c ‘python benchmarks/scripts/tf_cnn_benchmarks/tf_cnn_benchmarks.py --device=cpu --model=resnet50 --data_format=NWHC ’.
+
+You can replace the model with one of your choice supported by the TensorFlow benchmarks.
+
+
+
+Kubeflow Benchmarks
+===================
+
+The benchmark workload will run in a Kubernetes (k8s) container. We will use Kubeflow and deploy three nodes for this tutorial, to get a decent return.
```

Comment:
While I don't have an alternative phrase that I can suggest, I believe that `a decent return` can be replaced for something that better reflects why this particular deployment was selected for this tutorial. Better resource management? More output data?

Reply:
how about "...deploy three nodes for this tutorial to show resource management and get sufficient output data for evaluation."

Reply:
+ 1. I like this revision!